test/e2e/kustomize_test.go

```diff
@@ -138,3 +141,71 @@ func TestKustomizeDeclarativeInvalidApp(t *testing.T) {
 		Expect(SyncStatusIs(SyncStatusCodeUnknown)).
 		Expect(Condition(ApplicationConditionComparisonError, "invalid-kustomize/does-not-exist.yaml: no such file or directory"))
 }
+
+func TestKustomizeBuildOptionsNotFail(t *testing.T) {
+	// First test that the inputted build options are actually passed to `kustomize build` by creating an error, then
+	// test that acceptable parameters do not cause an error.
+	Given(t).
```

Comment:
how come we don't skip this test like below?

Reply:
This test uses build options that are present on all versions of Kustomize (i.e. `-v 6 --logtostderr`). The following test uses build options only present on Kustomize version 2.1.0+. With this, I wanted to test the functionality regardless of what version of Kustomize is available.